packages/core-ui/lib/buttons/index.tsx

```diff
@@ -0,0 +1,111 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
```

Comment:
I am getting the inkling that this entire file can be automatically built in the rollup process. We then do not need to update this file at all when new buttons are introduced.

I have began exploring custom rollup plugins with the [`rollup-plugin-remove-lines`](https://github.com/paritytech/polkadot-cloud/blob/main/packages/core-ui/plugins/rollup-plugin-remove-lines.js) file, which simply goes through the library and removes the provided lines if found. I have used it in `core-ui` to remove broken CSS imports. It is called at the beginning of the pipeline at the `writeBundle` phase.

I've also [created an issue](https://github.com/paritytech/polkadot-cloud/issues/286) to automatically generate `lib/index.tsx` so we don't have to manually input components.

All for different PRs, but a direction we should definitely head in.

Reply:
My only concern is if there should be internal components that we prefer not to export automatically through this automated process; But lets move the conversation in the newly open PR